zscaler_connector.py

```diff
@@ -64,7 +64,7 @@ def _get_error_message_from_exception(self, e):
                 elif len(e.args) == 1:
                     error_msg = e.args[0]
         except Exception:
-            pass
+            self.debug_print("Error occurred while getting message from response")
```

Comment:
Should we include the error message in the print? This should get us to the error faster.